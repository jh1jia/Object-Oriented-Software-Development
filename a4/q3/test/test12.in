test12.txt count z count 1
