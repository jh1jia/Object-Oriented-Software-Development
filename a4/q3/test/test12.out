12345678920345263748596107118129202132223242526272829303143233343536373839404154243444546474849505165253545556575859606176263646566676869
